ui/app/mainui/activitycenter/popups/ActivityCenterPopup.qml

```diff
@@ -369,6 +370,14 @@ Popup {
             notification: parent.notification
             store: root.store
             activityCenterStore: root.activityCenterStore
+            onReadMoreClicked: {
+                root.close()
+                Global.openPopup(newsMessagePopup, {
```

Comment:
I would recommend to avoid `Global.openPopup` and keep it local, e.g. by using `Component.createComponent` combined with `destroyOnClose` or by delcaring `NewsMessagePopup` without `Component` wrapper and call just `open`.

Reply:
I move the popup to Popups.qml in the next PR since that popup is used by multiple components, so that problem gets fixed by itself